graphium/trainer/losses.py

```diff
@@ -52,22 +52,40 @@ def __init__(
         self.alpha = alpha
         self.softmax = torch.nn.Softmax(dim=1)
 
-    def forward(self, input: Tensor, target: Tensor) -> Tensor:
+    def forward(self, input: Tensor, target: Tensor, nan_targets: Tensor = None) -> Tensor:
```

Comment:
Do the other loss functions not take a `nan_targets` argument?

Reply:
I think the other loss functions do not need `nan_targets` inside forward, it can be applied after in `ipu_losses.py`.